### PR TITLE
tests: Add test for CAS startup on reboot

### PR DESCRIPTION
**Pull request by robertbaldyga**
Requires: https://github.com/Open-CAS/test-framework/pull/93
Requires: https://github.com/Open-CAS/test-framework/pull/94

Signed-off-by: Robert Baldyga <robert.baldyga@intel.com>

### CODE REVIEW

File: test/functional/tests/startup/test_startup.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm
+from api.cas import init_config
```
Comment (micrakow) [inline test/functional/tests/startup/test_startup.py]: could be merged?

File: test/functional/tests/startup/test_startup.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        TestRun.executor.reboot()
+
+    with TestRun.step("Check if cache is started"):
+        caches = list(get_caches())
+        assert len(caches) == 1, f"Expected one cache, got {len(caches)}!"
+        assert caches[0].cache_id == cache.cache_id, "Invalid cache id!"
+
+    with TestRun.step("Check if core is added"):
+        cores = list(get_cores(cache.cache_id))
+        assert len(cores) == 1, f"Expected one core, got {len(cores)}!"
```
Comment (micrakow) [inline test/functional/tests/startup/test_startup.py]: general - shouldn't TestRun.fail be used?

Comment (robertbaldyga) [reply test/functional/tests/startup/test_startup.py]: Done.

File: test/functional/tests/startup/test_startup.py (hunk 1 above)
Comment (robertbaldyga) [reply test/functional/tests/startup/test_startup.py]: Done.

**Comment by robertbaldyga** [comment]
retest this please

**Pull request merged**